prov/rxm/src/rxm_ep.c

```diff
@@ -1011,10 +1150,10 @@ ssize_t rxm_ep_barrier2(struct fid_ep *ep, fi_addr_t coll_addr,
 	struct rxm_coll_buf *req;
 	ssize_t ret;
 
-        rxm_ep = container_of(ep, struct rxm_ep, util_ep.ep_fid.fid);
+	rxm_ep = container_of(ep, struct rxm_ep, util_ep.ep_fid.fid);
 
 	ret = rxm_ep_init_coll_req(rxm_ep, FI_BARRIER, flags, context,
-				   &req, &coll_ep);
+				   &req, &coll_ep, &coll_addr);
```

Comment:
`coll_addr` is supposed to be an input. Here a new return value is used instead. This looks very hacky. Same for other collective calls.

Reply:
Done